### DISCUSSION TITLE
q-select repeat the selected value in all rows q-table

### DISCUSSION THREAD

**Discussion by wea141619** [quasarframework/quasar#10925] [General - Components / Directives / etc]
Hello community I need your help, I  add a q-select into a q-table in the v-slot:body-cell, the problem I have is when I select the value of any of the rows of q-table the same value is selected in the all rows q-select.   How I can handle one value per q-select 

**Here my code**

<template v-slot:body-cell-impresora="props">
            <q-td :props="props">
              <q-select
                filled
                :options="options"
                map-options
                v-model="editedItem[props.row]"
                emit-value
                option-value="Nombre"
                clearable
                option-label="Nombre"
                label="Seleccionar"
              />
            </q-td>
          </template>

**declaration of the data**
 editedItem: [
        {
          Tipo: "",
          Nombre: "",
          Impresora: ""
        }
      ],

![image](https://user-images.githubusercontent.com/44841569/136496164-26e8c711-f5db-49ab-88e8-bbffe279517a.png)

thaks for any help

**Comment by metalsadman** [comment]
show some code please.

**Reply by wea141619** [reply]
Hello friend thanks for your time, here is the vue file with all the code
[FormatoImpresion.txt](https://github.com/quasarframework/quasar/files/7311903/FormatoImpresion.txt)

**Comment by ontwikkelfabriek** [comment]
@wea141619 

Please create a **codepen.io** for better support or at least use  code highlighting:
https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Default Quasar V2 codepen:
https://codepen.io/rstoenescu/pen/xxEvBLR

**Reply by wea141619** [reply]
Ok I will do thanks.
